### PR TITLE
Fetch Raw Data from activation and demog flows

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

**Comment by lukechurch** [comment]
How much of this is different? Anything that you'd like me to look at specifically or mostly just a copy into this repo?

**Comment by as2388** [comment]
This is all copy and paste except for configuration

File: pipeline_config.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "RapidProKeyRemappings": [
+
+  ],
+  "ProjectStartDate": "2000-01-01T00:00:00+03:00",
+  "ProjectEndDate": "2100-01-01T24:00:00+03:00",
+  "FilterTestMessages": false,
+  "DriveUpload": {
+    "DriveCredentialsFileURL": "gs://avf-credentials/pipeline-runner-service-acct-avf-data-core-64cc71459fe7.json",
+    "ProductionUploadPath": "ahadi_analysis_outputs/csap_s02_production.csv",
```
Comment (lukechurch) [inline pipeline_config.json:28]: As discuss `csap_s02_`

**Review by lukechurch — APPROVED** [review]
Thanks @as2388!

File: pipeline_config.json (hunk 1 above)
Comment (as2388) [reply pipeline_config.json:28]: fixed in a future PR

**Pull request merged**